hax/treemakers/corrections.py

```diff
@@ -87,4 +168,19 @@ def extract_data(self, event):
         result['cs2_top'] = s2.area * s2.area_fraction_top * s2_top_correction
         result['cs2_bottom'] = s2.area * (1.0 - s2.area_fraction_top) * s2_bottom_correction
 
```

Comment:
where is cs2 variable after this change? If we don't have cs2, the blinding cut will fail